### PR TITLE
feat(logging): add error logging on disputes and slow relay. Decode OO ancillary data

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

This PR does two things:
1) Generates `error` level log on disputes and slow relays.
2) fixes the OO bot to decode encoded ancillary data.

### CODE REVIEW

File: packages/insured-bridge-relayer/src/Relayer.ts [hunk 1]
```diff
@@ -559,7 +567,8 @@ export class Relayer {
   private async _sendTransaction(
     transaction: TransactionType,
     message: string,
-    mrkdwn: string
+    mrkdwn: string,
+    level = "info"
```
Comment (chrismaree) [inline packages/insured-bridge-relayer/src/Relayer.ts:571]: level defaults to `info` (what it was before). else, you can define a `level` that will be set on message emmited. we simply append "error" onto the `level` on a transaction.

File: packages/monitors/src/OptimisticOracleContractMonitor.js [hunk 2]
```diff
@@ -285,6 +290,15 @@ class OptimisticOracleContractMonitor {
   _isFundingRateIdentifier(identifier) {
     return identifier.toLowerCase().endsWith("_fr");
   }
+
+  _formatAncillaryData(ancillaryData) {
```
Comment (chrismaree) [inline packages/monitors/src/OptimisticOracleContractMonitor.js:294]: simply decode ancillary and append.

Comment (nicholaspai) [reply packages/monitors/src/OptimisticOracleContractMonitor.js:294]: +1

**Review by nicholaspai — APPROVED** [review]
lgtm

**Pull request merged**